htdocs/js/functions.js

```diff
@@ -196,8 +196,18 @@ vz.parseUrlParams = function() {
 					// disable automatic refresh
 					vz.options.refresh = false;
```

Comment:
My use case: I have a gas flow sensor, for which I would like to display the hourly consumption over the last x days, while also being able to display the current electric consumption over the last x minutes in another iframe. Currently it's not really possible to do this. 

One solution would be an additional option `interval`, which works the same as `from`, but doesn't disable the refresh and ignores the `to` parameter.

What I would propose though is to only disable autorefresh if `key=='to'`, as this would allow me to specify different intervals for viewing. If you still want auto refresh disabled (which doesn't make sense to me), you could specify `to="now"`. 

Does it sound good to you? Then I will append it to this commit.